File: DVLD_Project/UserInfo.vb
Imports BussnessLayerForDVLD
Imports System
Imports System.Data
Imports System.Windows.Forms

Namespace DVLD_Project
    Public Partial Class UserInfo
        Inherits UserControl
        Private _UserID As Integer = 0, _PersonID As Integer = 0
        Public Sub New()
            InitializeComponent()
        End Sub

        Public Sub New(userID As Integer)
            InitializeComponent()
            _UserID = userID
            Dim dt = clsBussnesLayer.FindUser(_UserID)
            If dt IsNot Nothing Then
                For Each dr As DataRow In dt.Rows
                    lblUesrName.Text = dr("UserName").ToString()
                    lblUserID.Text = dr("UserID").ToString()
                    lblActive.Text = dr("IsActive").ToString()
                    _PersonID = Convert.ToInt32(dr("PersonID"))
                Next
            End If
        End Sub

        Private Sub UserInfo_Load(sender As Object, e As EventArgs)

        End Sub

        Private Sub groupBox1_Enter(sender As Object, e As EventArgs)

        End Sub

        Private Sub linkLabel1_LinkClicked(sender As Object, e As LinkLabelLinkClickedEventArgs)
            Dim form As Form9 = New Form9(_UserID)
            form.ShowDialog()
        End Sub

        Public Function getPersonID() As Integer
            Return _PersonID
        End Function

    End Class
End Namespace

File: DVLD_Project/UserInfo.cs
using BussnessLayerForDVLD;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace DVLD_Project
{
    public partial class UserInfo : UserControl
    {
        int _UserID = 0,_PersonID=0;
        public UserInfo()
        {
            InitializeComponent();
        }

        public UserInfo(int userID)
        {
            InitializeComponent();
            _UserID = userID;
            DataTable dt = clsBussnesLayer.FindUser(_UserID);
            if(dt != null ) 
            foreach (DataRow dr in dt.Rows)
            {
                lblUesrName.Text = dr["UserName"].ToString();
                lblUserID.Text = dr["UserID"].ToString();
                lblActive.Text = dr["IsActive"].ToString();
                    _PersonID = Convert.ToInt32(dr["PersonID"]);
            }
        }

        private void UserInfo_Load(object sender, EventArgs e)
        {

        }

        private void groupBox1_Enter(object sender, EventArgs e)
        {

        }

        private void linkLabel1_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
        {
            Form9 form=new Form9(_UserID);
            form.ShowDialog();
        }

        public int getPersonID()
        {
            return _PersonID;
        }
        
    }
}
